runtime/tests/CMakeLists.txt

```diff
@@ -3,9 +3,14 @@ Include(FetchContent)
 FetchContent_Declare(
     Catch2
     GIT_REPOSITORY https://github.com/catchorg/Catch2.git
-    GIT_TAG        v2.13.9
+    GIT_TAG        v3.8.0
```

Comment:
Does Catalyst support Catch3 test syntax?

Reply:
I upgraded it yes :) It's not much different, there is a guide here: https://github.com/catchorg/Catch2/blob/v3.8.0/docs/migrate-v2-to-v3.md

Reply:
Nice work